### ISSUE TITLE
GeotechnicalUnit

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#22]
Source proposition : IFC Tunnel GeoSubgroup

A surface or a volume in which the mechanical behaviour and other design-relevant characteristics are
characterized using the same geotechnical parameters values. Several alternative classifications
(=GeotechModels) can be required in a project for different design tasks.

**Comment by mbeaufils** [comment]
Suggestion to replace "a surface or a volume" by "a body", like in GeoSciML to not restrict representations. eg. along a borehole that would be represent as a line, the geotechnical unit will be represented as a line.

**Comment by Alexis-SERIEYS** [comment]
> Suggestion to replace "a surface or a volume" by "a body", like in GeoSciML to not restrict representations. eg. along a borehole that would be represent as a line, the geotechnical unit will be represented as a line.

I don't agree, because as I understood, the object "Discrete Discontinuity" was a specification of the object "Geotechnical Unit". Wich is a surface with geotechnical parameters. So the term "surface" seems to be important

**Comment by Alexis-SERIEYS** [comment]
Also, as a general comment, I would say that a "Geotechnical Unit" should not necessarily be a specification of a "Geologic Unit".

I agree that the mechanics parameters are highly conditioned by the geology (lithology, age, etc ...). But, the goetechnic model is often established using simplification or regroupement and could not strictly repect the boundaries of geological model. 

So,  I think it should be possible to link a "Geological Unit" and a "Geotechnical Unit", but these two object should be a the same level of hierarchy.

**Comment by BLEHELLO** [comment]
I agree with Alexis, geotechnical unit, geological unit and even hydrogeological unit have ti be at the same level hierarchy with link between them

**Comment by Didymograptus** [comment]
> Suggestion to replace "a surface or a volume" by "a body", like in GeoSciML to not restrict representations. eg. along a borehole that would be represent as a line, the geotechnical unit will be represented as a line.

Agreed, we are straying into the representation. We already have terms like mass, volume, surface, body etc. floating around. We should concentrate on the logical model at the moment and the properties associated with it. Unit is fine.

**Comment by neilchadwick-dg** [comment]
Copied from my comment in #7, but more relevant here...

It is the geotechnicalunit that will be used in analysis/design. This is normally derived from the geologic units. The classification system may be the same, or it may differ to suit to ground and/or analysis method. The extents are likely to differ to take account of uncertainty, i.e. assessing 'characteristic' extends. Parameters for design can be assigned.

A geological model with geological units may give birth to several different geotechnical models with different geotechnical units (or at least different parameters).

**Comment by neilchadwick-dg** [comment]
At the meeting yesterday, it became clear that I (and possibly some others) had interpreted things a slightly different way. The discussion helped to clarify matters. For the record, and for the benefit of those not present, I will try to summarise what came up at the meeting (hope I've got it right - please correct me if not). I will then add some further observations that I hope will help going forward.

When talking about geotechnical units and models above, I admit that I was thinking more about 'Book C' (design), and had not realised that the object proposed in this Issue was for Book B (interpretation). I understand that IFC Tunnel propose the term 'Geotechnical synthesis model' for the Book C version. I'm ok with that (although I will comment further below).

We talked a little about what a Book B geotechnical model might look like. Consensus was that the classification of units should be 'geotechnical', looking forward to design, but both parameters and extents of units would most likely be ranges, not specific values. This information could then be used as input for the 'Book C' geotechnical synthesis model (which can only be confirmed when more is known about the design) which would (should) have specific values assigned.

This workflow does not happen all of the time, but that is ok. We are not dictating what models are used, only accommodating those that may be needed. However, it is worth noting that this workflow ties in well with EC7, i.e. Book A + Book B = GIR, whereas the work for Book C is covered by the GDR.

On the difference between a geological model and a geotechnical model at Book B stage: I see the main difference being classification. The geological model concentrates on the geology (including nature, origin, past processes), whereas the geotechnical model is more focused on the expected mechanical behaviour of the ground. It is possible, but not certain, that the boundaries shown in each model may be the same (that is something for the project to decide). However, as we discussed it is common for a geotechnical unit to be assigned that combines several adjacent geological units, especially in soft ground. Conversely, a single geological unit may include layers which sometimes leads to it being split up into different geotechnical units.

Further observations...

- Whilst I am ok with the distinction between Book B geotechnical model and Book C geotechnical synthesis model, I would be ok we just called both 'geotechnical models'.  The specific intended/permitted use cases for each model will always need to be defined anyway. The Book B model will just have a different, broader use case.
- On a similar theme, I see no harm in geotechnical units being part of both the Book B and Book C models. In fact, I believe this should be the case. They represent the same thing (provided we get the definition right), just used in a different context.
- As I said in the meeting, I don't think we focus too much on where in the Book A/B/C process a particular object/thing may appear. It is the nature of the object that matters most. A geological model/unit  is a geological model/unit, regardless of where in the process it appears. Yes, we should consider common processes, and this helps our thinking, but I believe that the conceptual model developed should be process agnostic as far as possible.

**Comment by PieGARNIER** [comment]
Neil, I have just one comment: I believe GeotechnicalUnit could differ from Book B to Book C. Book B would be the global interpretation of geotechnical conditions (representative values), but Book C, in reference to Book B, would contain Design values (the one sent to structural engineers for example). Since Book C is related to the project, it could differ from book B (for example to design a shallow foundation, design values are related to the geometry of the foundation).

**Comment by mbeaufils** [comment]
Having a deeper look to the [type of GeologicUnit](https://inspire.ec.europa.eu/codelist/GeologicUnitTypeValue/geologicUnit) as defined by CGI and reused in the INSPIRE Directive.

[LithogeneticUnit](https://inspire.ec.europa.eu/codelist/GeologicUnitTypeValue/lithogeneticUnit) seems interesting for our purpose. Would it be synonymous to our Geotechnical Unit for you?

**Comment by mbeaufils** [comment]
> > Geologic unit defined by genesis. The genesis is manifested by material properties, but the material is not the defining property.
> 
> The geotechnical unit **IS** defined by its material properties.

@Didymograptus : yes. The French translation is less ambiguous with the use of the word material "Geological unit defined by its genesis. The genesis is manifested by the physical properties, but the material is not the defining property.". I understand here that a Lithogenenic unit can be made of different materials and the focus is on the "global" physical properties of this unit.

**Comment by SylvieBretelle** [comment]
I agree with the above. a geotechnical unit can have the same boundaries as a geological unit, can be part of a geological unit or can be assembly of different geological units. The very first definition is correct to me:
"A body in which the mechanical behaviour and other design-relevant characteristics are
characterized using the same geotechnical parameters values. "
this definition is valid for both Book B and book C.

**Comment by neilchadwick-dg** [comment]
For information, the current Draft of the upcoming revision to Eurocode 7 (Part 1) includes the following definitions:

3.1.6.4
**Geotechnical Design Model**
conceptual representation of the site derived from the ground model for the verification of each
appropriate design situation and limit state

3.1.6.5  
**geotechnical unit**
volume of ground or ground layer that is defined as a single material in a Geotechnical Design Model

3.1.6.6
**Ground Model**
site specific outline of the disposition and character of the ground and groundwater based on results from
ground investigations and other available data

The definition of geotechnical unit and Geotechnical Design Model are consistent with my own thoughts and with AGSi.

The EC7 use of *Ground Model* appears to align it with what I would call an observational model, i.e. the geological model, as I (and AGSi) prefer to use *Ground Model* as a parent term for any type of model. So be it!
